web/themes/custom/subtheme_videogame/src/components/field/_genrestoplay.scss

```diff
@@ -0,0 +1,37 @@
+.genres-to-play {
+  &__body {
+    margin-top: 3rem;
+  }
+
+  &__field-genre {
+    font-weight: bold;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+}
+
+.field__item {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0 1rem 0.5rem 0;
+  width: 120px;
+  height: 120px;
+
+  &:nth-of-type(1), &:nth-of-type(7) {
```

Comment:
I could set the same colors for these ones:
 &:nth-child(6n+1) {
    background-color: rgba($orange, 0.7);
  }  <--- The 1st and the 7th.

  &:nth-child(4n+2) {
    background-color: rgba($orange, 0.8);
  } <--- The 2nd and 6th.

But I can't find with this method only the 3rd and 5th because with "2n+3" the 7th gets the same color. :/

Reply:
what about 6n+3 and 6n+6? One row contains 6 elements, right?

Reply:
or if there are 7 elements in a row, you can use 7n+1, 7n+2 etc